### PR TITLE
added firebase config

### PR DESCRIPTION
**Pull request by wmm3636**
please review my changes

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
+import firebase from 'firebase';
```
Comment (juazsh) [inline App.js:2]: Have you run the app and tested if it's showing any erro.

File: App.js [hunk 2]
```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
```
Comment (juazsh) [inline App.js]: please follow the import statement rules and place required imports before ui imports

**Review by juazsh — CHANGES_REQUESTED** [review]

File: App.js (hunk 2 above)
Comment (wmm3636) [reply App.js]: fixed it

File: App.js (hunk 1 above)
Comment (wmm3636) [reply App.js:2]: yes I did

**Comment by wmm3636** [comment]
Please review it again

**Review by juazsh — APPROVED** [review]

**Pull request merged**